crud/create.js

```diff
@@ -0,0 +1,45 @@
+module.exports = (binding) => {
+    let columns = binding.fields.filter(f => !f.identity).map(f => `[${f.column}]`).join(', ');
+    return `
+    CREATE PROCEDURE ${binding.spName}
+        @data ${binding.tt} READONLY
+    AS
+    SET NOCOUNT ON
+    DECLARE @result ${binding.tt}
+    BEGIN TRY
+        INSERT INTO ${binding.name} (${columns})
+        OUTPUT INSERTED.* INTO @result

```

Comment:
Is `@result` really needed here? Is it needed because of history triggers?

Reply:
No praticular reason. Just thought that storing the data in a table type would be easier to return all the data as a named resultset at the end.